docs/config.rst

```diff
@@ -1,11 +1,606 @@
+
+To reference a return value on a previous task use the following:
+
+.. code-block:: yaml
+
+    ^^prior_task.return_value
+
+To discover what's available for ``<return_value>``, find the source code for an individual task.
+
+    Example: Let's examine how CumulusCI defines the standard ``upload_beta`` task in the universal ``cumulusci.yml`` file.
+
+.. code-block:: yaml
+
+    upload_beta:
+            description: Uploads a beta release of the metadata currently in the packaging org
+            class_path: cumulusci.tasks.salesforce.PackageUpload
+            group: Release Operations
+
+In order to see if anything is being set on ``self.return_values``, find the file where the class ``cumulusci.tasks.salesforce.PackageUpload`` is defined. A little digging yields that this class is defined in the file `package_upload.py<TODO>`_ and has a method called ``_set_return_values()``. `This method <https://github.com/SFDO-Tooling/CumulusCI/blob/3cad07ac1cecf438aaf087cdeff7b781a1fc74a1/cumulusci/tasks/salesforce/package_upload.py#L165>`_ sets ``self.return_values`` to a dictionary with the keys: ``version_number``, ``version_id``, and ``package_id``.
+
+Now look at the standard ``release_beta`` flow defined in the universal ``cumulusci.yml`` file:
+
+.. code-block:: yaml
+
+   release_beta:
+        description: Upload and release a beta version of the metadata currently in packaging
+        steps:
+            1:
+                task: upload_beta
+                options:
+                    name: Automated beta release
+            2:
+                task: github_release
+                options:
+                    version: ^^upload_beta.version_number
+            3:
+                task: github_release_notes
+                ignore_failure: True  # Attempt to generate release notes but don't fail build
+                options:
+                    link_pr: True
+                    publish: True
+                    tag: ^^github_release.tag_name
+                    include_empty: True
+                    version_id: ^^upload_beta.version_id
+            4:
+                task: github_master_to_feature
+
+This flow shows how subsequent tasks can reference the return values of a prior task. In this case, the ``github_release`` task uses the ``version_numer`` set by the ``upload_beta`` task as an option value with the ``^^upload_beta.version_number`` syntax. Similarly, the ``github_release_notes`` task uses the ``version_id`` set by the ``upload_beta`` task as an option value with the ``^^upload_beta.version_id`` syntax.
+
+
+
+Flow Configurations
+-------------------
+
+Flow configurations are defined under the ``flows`` section of your ``cumulusci.yml`` file. Each flow configuration defines a flow which can be run using the ``cci flow run`` command, or included in a flow step. With a few simple changes to this section, you can configure build automation functionality without ever touching Python code.
```

Comment:
```suggestion
Each flow configuration defined in the ``flows`` section of your ``cumulusci.yml`` file defines a flow that can be run using the ``cci flow run`` command, or included in a flow step. With a few simple changes to this section, you can configure build automation functionality without ever touching Python code.
```

Reply:
@Br4nd0R Is there another word for "defined" in this instance? I don't like using defined followed by defines.

Reply:
Agreed. Maybe: "Each flow configuration specified in the ``flows`` section of your..."?